### ISSUE TITLE
x/review/git-codereview: mail requires -trust=false

### ISSUE THREAD

**Issue by johanbrandhorst** [golang/go#56343]
### What version of Go are you using (`go version`)?

<pre>
$ go version
go version go1.19.2 linux/amd64
</pre>

### Does this issue reproduce with the latest release?

Yes, I ran `go install golang.org/x/review/git-codereview@latest`

### What operating system and processor architecture are you using (`go env`)?

<details><summary><code>go env</code> Output</summary><br><pre>
$ go env
GO111MODULE=""
GOARCH="amd64"
GOBIN=""
GOCACHE="/home/johan/.cache/go-build"
GOENV="/home/johan/.config/go/env"
GOEXE=""
GOEXPERIMENT=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GOINSECURE=""
GOMODCACHE="/home/johan/go/pkg/mod"
GONOPROXY=""
GONOSUMDB=""
GOOS="linux"
GOPATH="/home/johan/go"
GOPRIVATE=""
GOPROXY="https://proxy.golang.org,direct"
GOROOT="/nix/store/8s8mxpa681mbddak431k8nxg1ib5pw0g-go-1.19.2/share/go"
GOSUMDB="sum.golang.org"
GOTMPDIR=""
GOTOOLDIR="/nix/store/8s8mxpa681mbddak431k8nxg1ib5pw0g-go-1.19.2/share/go/pkg/tool/linux_amd64"
GOVCS=""
GOVERSION="go1.19.2"
GCCGO="gccgo"
GOAMD64="v1"
AR="ar"
CC="gcc"
CXX="g++"
CGO_ENABLED="1"
GOMOD="/dev/null"
GOWORK=""
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -Wl,--no-gc-sections -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build1025149787=/tmp/go-build -gno-record-gcc-switches"
</pre></details>

### What did you do?

I was following the contribution guide (https://go.dev/doc/contribute), and after running `git mail` to submit a change I got an unexpected error.

### What did you expect to see?

I expected for `git mail` to create a CL with my changes.

### What did you see instead?

```shell
$ git mail
remote: Processing changes: refs: 1, done    
To https://go.googlesource.com/go
 ! [remote rejected]       HEAD -> refs/for/master%l=Trust (label "Trust" is not a configured label)
error: failed to push some refs to 'https://go.googlesource.com/go'
(running: git push -q origin HEAD:refs/for/master%l=Trust)
git-codereview: exit status 1
```

Running `git mail -trust=false` fixes the issue, but I don't understand why this is required. Nothing in the contribution guide mentions this flag.

**Comment by cuonglm** [comment]
`-trust` is not used anymore: https://github.com/golang/review/commit/f2105c4cf825fe09ece01c9d554f6c0a5bc06a03

Sounds like you want `@master` instead of `@latest`.

**Comment by mvdan** [comment]
Given that the majority of people will default to using "latest", perhaps we can aim to regularly tag x/review. Especially when breaking changes like these are made. cc @golang/release

**Comment by mvdan** [comment]
On second thought, the latest commit is already tagged. So I guess we already do this. I'm confused.

**Comment by dmitshur** [comment]
The -trust flag is obsolete and was removed in [CL 399117](https://go.dev/cl/399117), which is included in v1.1.0. Maybe the `git-codereview` binary you installed with `go install golang.org/x/review/git-codereview@latest` isn't in your PATH? What's the output `go version -m $(which git-codereview)`?

**Comment by johanbrandhorst** [comment]
> The -trust flag is obsolete and was removed in [CL 399117](https://go.dev/cl/399117), which is included in v1.1.0. Maybe the `git-codereview` binary you installed with `go install golang.org/x/review/git-codereview@latest` isn't in your PATH? What's the output `go version -m $(which git-codereview)`?

Oops, this does seem to be what happened here, thanks for the tip! False alarm 😄.
